lstchain/reco/dl0_to_dl1.py

```diff
@@ -43,7 +44,7 @@
 # Add option to use custom calibration
 
 custom = False
-cal = CameraCalibrator(extractor_name="NeighborPeakWindowSum")
+cal = CameraCalibrator(image_extractor=NeighborPeakWindowSum())
 
```

Comment:
Ah! Thanks for this, I had my ctapipe version outdated and this was driving me crazy because for some reason it worked but gave wrong hillas parameters... Nice!